doc/modules/ROOT/pages/graph-object.adoc

```diff
@@ -161,6 +162,129 @@ If multiple node dataframes are used, they need to contain distinct node ids acr
+----
+
+
+[TIP]
+====
+Combined with https://networkx.org/documentation/stable/reference/readwrite/index.html[NetworkX's functionality for reading various graph formats] one can easily load popular graph formats like edge list and GML into GDS.
+====
+
+[TIP]
+====
+Combined with https://networkx.org/documentation/stable/reference/generators.html[NetworkX's functionality for generating various kinds of graphs] one can easily load graph types popular in the literature into GDS, such as expanders, lollipop graphs, complete graphs, and more.
+====
+
+
+[[nx-schema]]
+=== NetworkX schema to GDS schema
+
+There are some rules as to how the NetworkX graph maps to the projected `Graph` in GDS.
+They follow principles similar to those of xref:graph-object#construct[], and are outlined in detail in this section.
+
+
+==== Node labels
+
+Node labels for the projected GDS graph are taken from attributes on `networkx.Graph` nodes.
+The values of node attribute key `labels` will dictate what labels nodes are given in the projection.
+These values should be either strings or lists of strings.
+
+Either all nodes of the `networkx.Graph` must have valid `labels` attributes, or they can be _completely_ left out from the graph.
+That is, a `networx.Graph` with no `labels` node attributes at all is also allowed.
+In this latter case, the nodes in the projected `Graph` will all have the node label `N`.
+
+
+==== Node properties
+
+Node properties for the projected GDS graph are taken from attributes on `networkx.Graph` nodes.
+The keys of the attributes will map to property names, and the allowed values must follow the regular guidelines for node property values in GDS.
+Please note though that the node attribute key `labels` is reserved for node labels (previous section) and will not translate to node properties in the projection.
+
+Nodes having the same values for the `labels` attribute must have the same set of node property keys.
+
+
+==== Relationship types
+
+Relationship types for the projected GDS graph are taken from attributes on `networkx.Graph` edges.
+The values of edge attribute key `relationshipType` will dictate what types relationships are given in the projection.
+These values should be either strings or omitted.
+
+Either all edges of the `networkx.Graph` must have valid `relationshipType` attributes, or they can be _completely_ left out from the graph.
+That is, a `networx.Graph` with no `relationshipType` edge attributes at all is also allowed.
+In this latter case, the relationships in the projected `Graph` will all have the relationship type `R`.
+
+
+==== Relationship properties
+
+Relationship properties for the projected GDS graph are taken from attributes on `networkx.Graph` edges.
+The keys of the attributes will map to property names, and the allowed values must follow the regular guidelines for relationship property values in GDS.
+That includes the limitation of having at most one property per relationship.
```

Comment:
This is not a limitation in GDS, a relationships can have more than one property projected.
However for algorithms one property needs to be selected.

Maybe we want to ease up on this restrictions?

Reply:
Fixed